src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java

```diff
@@ -422,4 +424,36 @@ private void startJnlpProc() throws Exception {
         });
     }
 
+    @Issue("JENKINS-26132")
+    @Test public void enclosingLabel() {
```

Comment:
Until https://github.com/jenkinsci/workflow-basic-steps-plugin/pull/1 is merged, it is hard to test nested labels. Could so with `parallel`, I suppose. But the heavier tests of that aspect are already in the upstream PR.